### PR TITLE
Update poms and other locations to 2.1.0-SNAPSHOT

### PR DESCRIPTION
**Pull request by ZacBlanco**

### CODE REVIEW

**Comment by ZacBlanco** [comment]
@madanadit 
@calvinjia

File: docs/_config.yml [hunk 1]
```diff
@@ -13,10 +13,10 @@ ALLUXIO_MASTER_VERSION_SHORT: master
 # For master branch, this should be "master"
 ALLUXIO_MAJOR_VERSION: master
 # The version string must match the version string part of the client jar path
-ALLUXIO_VERSION_STRING: 2.0.0-SNAPSHOT
+ALLUXIO_VERSION_STRING: 2.1.0-SNAPSHOT
```
Comment (madanadit) [inline docs/_config.yml]: @ZacBlanco update ALLUXIO_RELEASED_VERSION as well

Comment (ZacBlanco) [reply docs/_config.yml]: done

**Comment by madanadit** [comment]
alluxio-bot, merge this please.

**Pull request merged**